common/v2/components/Profile/Profile.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import { Address } from '@mycrypto/ui';
+import { truncate } from 'v2/utils';
+import { get3BoxProfile, ThreeBoxProfile } from 'v2/services';
+
+const AvatarImage = styled.img`
+  height: 100%;
+  width: 100%;
+  object-fit: cover;
+  border-radius: 50%;
+`;
+
+const Avatar = styled.div`
+  width: 60px;
+  height: 60px;
+`;
+
+interface Props {
+  address: string;
+  label?: string;
+}
+
+interface State {
+  profile: ThreeBoxProfile;
+}
+
+export default class Profile extends Component<Props, State> {
```

Comment:
Prefer Function components over class components.
This facilitates reusing hooks.